Public/Src/Engine/Scheduler/PipExecutor.cs

```diff
@@ -1835,9 +1835,9 @@ private static void ReportFileAccesses(ExecutionResult processExecutionResult, F
                                 }
 
                                 var pathSet = maybePathSet.Value;
-                                (ObservedInputProcessingResult observedInputProcessingResult, StrongContentFingerprint? strongContentFingerPrint, ObservedPathSet pathSetUsed, ContentHash pathSetHashUsed)
+                                (bool succeeded, ObservedInputProcessingResult observedInputProcessingResult, StrongContentFingerprint? strongContentFingerprint, ObservedPathSet pathSetUsed, ContentHash pathSetHashUsed)

```

Comment:
>succeeded [](start = 38, length = 9)

I dont' see this variable used...

Reply:
It is sneakily hiding on the right side of line 1850

---
In reply to: [273725711](https://github.com/Microsoft/BuildXL/pull/114#discussion_r273725711) [](ancestors = 273725711)